Extend documentation center line (resolves #225)

carsten-kuebler opened the pull request:
This PR address resolves #225

Enhance the documentation of center line.

CarloVanDriestenBMW commented:
Should we consider removing Centerlines from the GT and add it to SensorData only? Then it would be model dependend how the CL is calculated?

HendrikAmelunxen commented:
I agree to this. The "virtual" centerline information (= "the line that a typical vehicle is supposed to follow") does not fit to the groundtruth information.

AndreHildebrandt commented:
I also agree with the removal of the centerline. I already gave some reasons (#225 ). For the moment I don't see the benefit, but the effort of generating and transmitting point sequences, where the validity is in the best case questionable.

carsten-kuebler commented:
@LudwigFriedmannBMW I changed the documentation. I moved the information to a note. Please note that in case of discontinuities in the roadway width, the center line may also have discontinuities. The center line was not intended to be an exact geometric center line.

LudwigFriedmannBMW commented on osi_lane.proto:
```diff
@@ -54,56 +54,58 @@ message Lane
+        // \image html line_approximation_error.png "Approximation error as
+        // green line."
+        //
+        // \note The center line is the line that a typical vehicle follows more
```
I would change this paragraph as follows:

The center line is the line that a typical vehicle follows more or less (depending on the situation, a little more to the left or right of the center line). The intended direction of travel on the lane is given by the direction defined by the sequence of points forming the center line.

carsten-kuebler commented:
@LudwigFriedmannBMW. I also refer to centerline_is_driving_direction.

LudwigFriedmannBMW commented:
Hello @carsten-kuebler, can you please resolve the conflicts?

carsten-kuebler commented:
@LudwigFriedmannBMW merged.
